cmd/cloud/deployment.go

```diff
@@ -51,6 +55,12 @@ var (
 		# Update a deployment variables from a file
 		$ astro deployment variable update --deployment-id <deployment-id> --load --env .env.my-deployment
 		`
+	wQueueConcurrency int
+	wQueueMin         int
```

Comment:
We should name these more closely to the variable names in Astrohub, for consistency. At least `wQueueMinWorkerCount` or `wQueueMinWorkers`. Same comment goes for max worker.

Reply:
@frank-ye Will chance it. Thanks!